src/themes/default/components/core/blocks/Footer/Newsletter.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <div class="newsletter py25 px15 bg-cl-secondary" v-if="!isSubscribed">
+  <div class="newsletter py25 px15 bg-cl-secondary" v-show="!isSubscribed">
```

Comment:
please test it carefully if any hydration issues are appearing, I remember we had some issues here with it

Reply:
actually I haven't noticed anything. I used this `show` instead of `if` as it solves the problem with closing dialog...